src/client/src/components/Quiz/Quiz.jsx

```diff
@@ -1,12 +1,28 @@
 import "./quiz.css";
 import BasicQuestion from "./BasicQuestion/BasicQuestion";
 import ProgressBar from "./ProgressBar/ProgressBar";
+import { useEffect, useState } from "react";
+
+export default function Quiz({ questions, removeQuestion }) {
```

Comment:
removeQuestion -- from the batch of questions? Because it is answered. Maybe we can specify it in the name like removeAnsweredQuestion or markQuestionAsAnswered / Skipped